lte/gateway/python/magma/mobilityd/ip_address_man.py

```diff
@@ -266,7 +269,7 @@ def list_allocated_ips(self, ipblock: ip_network) -> List[ip_address]:
             res = self.ip_allocator.list_allocated_ips(ipblock)
         return res
 
-    def alloc_ip_address(self, sid: str) -> ip_address:
+    def alloc_ip_address(self, sid: str) -> (ip_address, int):
```

Comment:
nit: let's use Tuple[ip_address, int] to define the typing of what this function return